backend/src/main/java/com/bakdata/conquery/models/index/MapInternToExternMapper.java

```diff
@@ -79,7 +74,7 @@ public boolean initialized() {
 
 	@Override
 	public String external(String internalValue) {
-		return int2ext.getOrDefault(internalValue, "");
+		return int2ext.getOrDefault(internalValue, internalValue); //TODO @ToniTub this seems like the better option?
```

Comment:
Maybe :D Bisher hatten wir quasi redundante Ausgabewerte, einen der eine ID / einen internen Wert ausgegeben hat und einen der eine ID gemapped ausgegeben hat

Reply:
Deswegen war es bisher blank um die ausgabe in einer Spalte einheitlich zu haben

Reply:
Hat der mappende Ausgabewert wenn er nicht mappen konnte dann blank ausgegeben? Das fände ich ungeschickt, weil das gut passieren kann durch Referenzen die nicht aktuell gehalten sind.

Reply:
Das stimmt, dann geben wir erstmal den internen Wert aus